src/renderTask.js

```diff
@@ -18,11 +16,37 @@ function render() {
     checkbox.id = task.id;
     checkbox.classList.add('custom-checkbox');
     checkbox.checked = task.complete;
-    const label = taskItem.querySelector('label');
-    label.htmlFor = task.id;
-    label.append(task.description);
+    const textarea = taskItem.querySelector('textarea');
+    textarea.append(task.description);
+    const trashBtn = taskItem.querySelector('.delete');
+    trashBtn.id = task.id;
     tasksContainer.appendChild(taskItem);
   });
 }
 
-export { render, tasks, tasksContainer };
\ No newline at end of file
+function focus(e) {
+  const clicked = e.target;
```

Comment:
- [ ] Here as well and all the places. I would recommend using `es6` syntax as it is a project requirement that helps you in writing more clean code using ES6 syntax.